rocAL/rocAL/source/augmentations/geometry_augmentations/node_resize.cpp

```diff
@@ -62,6 +62,11 @@ void ResizeNode::update_node() {
     std::vector<uint32_t> src_h_dims, src_w_dims;
     src_w_dims = _inputs[0]->info().get_roi_width_vec();
     src_h_dims = _inputs[0]->info().get_roi_height_vec();
+    if (_scaling_mode == RocalResizeScalingMode::ROCAL_SCALING_MODE_NOT_SMALLER) {
+        // Using original width and height instead of the decoded width and height for computing resize shorter dimensions
+        src_w_dims = _inputs[0]->info().get_original_roi_width_vec();
+        src_h_dims = _inputs[0]->info().get_original_roi_height_vec();
+    }
```

Comment:
For all modes we should calculate using original width and height ryt? 
Can we change this to default?

Reply:
Resolved in the latest commit